app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HelloRailsReact</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
+    <%= javascript_include_tag "application", "data-turbo-track": "reload", defer: true %>
+  </head>
+
+  <body>
+    <%= yield %>
+  </body>
```

Comment:
As required for the project and as mentioned the given **[tutorial](https://betterprogramming.pub/react-with-rails-2022-bd28e1fcd355#ad0a)**, kindly ensure that you have created a static view that will be the root of your app. It should include only `<div id="root">` The rest should be done with React and Redux. i.e.
![Screenshot from 2023-07-20 03-37-05](https://github.com/OsvaldoBC/hello-rails-react/assets/83042742/549fb6ed-17f9-40e7-8b4e-6e0a898742a7)